bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -1115,6 +1095,8 @@ public void operationComplete(int rc, LedgerMetadata result) {
         LedgerMetadataCmd() {
             super(CMD_LEDGERMETADATA);
             lOpts.addOption("l", "ledgerid", true, "Ledger ID");
+            lOpts.addOption("dumptofile", true, "Dump metadata for ledger, to a file");
```

Comment:
shall we add validation that `dumptofile` and `restorefromfile` are mutually exclusive

Reply:
added